.github/workflows/publish-maven-snapshots.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+        - main
+        - '[0-9]+.[0-9]+'
+        - '[0-9]+.x'
+
+jobs:
+  build-and-publish-snapshots:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
```

Comment:
```suggestion
      - name: Set up JDK 8
```

Reply:
Scratch that please, we use JDK 11 to build

Reply:
FYI - I didn't make any changes to this job, its merged directly from what is in 1.3.  Going to keep this unchanged